java-api-core/src/com/kinvey/java/store/requests/data/save/SaveListBatchRequest.kt

```diff
@@ -87,7 +87,7 @@ class SaveListBatchRequest<T : GenericJson>(
         val batchSaveErrors = ArrayList<KinveyBatchInsertError>()
 
         val updateResponse = updateObjects(updateList)
-        saveEntities.addAll(updateResponse.entities)
+        saveEntities.addAll(updateResponse.entities as Collection<T>)
```

Comment:
is this cast required?

Reply:
Added additional conversion for entities list to make it immutable.